protobuf/fix_proto_src.py

```diff
@@ -0,0 +1,20 @@
+"""
+This just does:
+sed -i 's/->base/->__base/g' <file>
+sed -i 's/ProtobufCMessage base;/ProtobufCMessage __base;/g' <file>
+"""
```

Comment:
Why can't we just have a shell script which does this? Why do we need Python? Other than the fact that it is easy to write, I don't see any other reason. But again it's just a one time effort, so we might as well not force people to run Python when this can be also done with bash.

Reply:
I don't know. I did just copy the files in this case.